src/plugins/hiddenRows/test/publicAPI.e2e.js

```diff
@@ -13,34 +13,126 @@ describe('HiddenRows', () => {
   });
 
   describe('public API', () => {
-    it('should hide row after calling the hiderow method', () => {
-      handsontable({
-        data: Handsontable.helper.createSpreadsheetData(3, 3),
-        hiddenRows: true,
+    describe('hideRow()', () => {
```

Comment:
Could you add the same changes within the file for the `HiddenColumns` plugin?

Reply:
I'll do that in a separate issue.